src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', (e) => {
+    resolve('Promise was resolved!');
+  });
+});
+
+promise1.then((value) => {
+  const message = document.createElement('div');
+
+  message.classList.add('message');
+  message.textContent = value;
+
+  document.body.appendChild(message);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promise2.catch((value) => {
+  const message = document.createElement('div');
+
+  message.classList.add('message');
+  message.classList.add('error-message');
+  message.textContent = value;
```

Comment:
The 'value' in the catch block is an Error object, not a string. To display the error message, you should use 'value.message' instead of 'value'.